NEWS.md

```diff
@@ -1,101 +1,133 @@
-3.32.1
+3.33.1
 ======
 
 ## Highlights
 
- * `mongodb()`: add `workers()` support (multi-threaded connection pooling)
-   
-   The MongoDB driver now supports the `workers()` option, which specifies the
-   number of parallel workers to be used.
-   Workers are based on the connection pooling feature of the MongoDB C library.
-   
-   This increases the throughput of the MongoDB destination driver.
-   
-   Example:
-   
+ * MQTT destination
+
+   The new `mqtt()` destination can be used to publish messages using the MQTT protocol.
+   Currently MQTT 3.1.1 and 3.1 are supported, 5.0 and SSL support will be added later.
```

Comment:
I do not think that we have to mention 5.0 and SSL support, this reads like we commit to support those. I would simply either not mention them, or note that those are not yet supported.

Reply:
Okay, I added a note instead.

Reply:
Thanks.